apps/meteor/tests/end-to-end/api/settings.ts

```diff
@@ -276,26 +276,34 @@ describe('[Settings]', () => {
 	});
 
 	(IS_EE ? describe : describe.skip)('/audit.settings', () => {
-		const formatDate = (date: Date) => date.toISOString().slice(0, 10).replace(/-/g, '/');
+		const formatDate = (date: Date) => date.toISOString();
+		let startDate: Date;
+		let endDate: Date;
+
+		before(async () => {
+			// Change a few settings so we can test filtering
+			startDate = new Date();
+			await Promise.all([updateSetting('Accounts_AllowAnonymousRead', true), updateSetting('Accounts_AllowDeleteOwnAccount', true)]);
```

Comment:
changing the setting and then changing it back right afterwards?

Reply:
This is just to trigger logs so that I can properly test the filters. That's why I revert them immediately, I don't need these settings, it's just to add log records to the server-events collection.